### PR TITLE
Documentation update - Closes #699

### LINKED ISSUE #699: Improve documentation
### Description

Add the following informations to the documentation pages:

- Describe `SERVICE_CORE_REDIS_VOLATILE` and explain how different is it from other Redis instances.
- Add the recommendation to start Lisk Service once the Core node is in sync.

### Acceptance Criteria

- Documentation is updated with the points above

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?
This PR resolves #699

### How was it solved?
- [x] Updated Readme
- [x] Updated configuration

### How was it tested?
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ export ENABLE_FEE_ESTIMATOR_QUICK="true"
 
 When running a local instance of Lisk Core and Lisk Service that is run by Docker, then the following variable needs to be set: `LISK_CORE_WS="ws://host.docker.internal:8080"`.
 
+It is strongly recommended to sync Lisk Core with the network **before** starting Lisk Service.
```
Comment (sameersubudhi) [inline README.md:106]:
```suggestion
It is strongly recommended to synchronize your Lisk Core node with the network **before** starting the Lisk Service.
```

File: docs/config_options.md [hunk 2]
```diff
@@ -90,10 +90,16 @@ LISK_CORE_HTTP=https://mainnet.lisk.com # Lisk Core HTTP URL
...
 ## Lisk Service Core
 
-# Local Redis cache for Lisk microservice
+# Local Redis cache with persistency for Core microservice
+# Refer to the /docker/redis/redis.persistent.conf for more details
 # Note: SERVICE_BROKER uses a different DB
 SERVICE_CORE_REDIS=redis://localhost:6379/1
 
+# Local Redis LRU cache for Core microservice
+# This should be a different instance on production
```
Comment (sameersubudhi) [inline docs/config_options.md:99]:
```suggestion
# This should be a separate instance in the production
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Please find a couple of suggestions.

**Review by chris529 — APPROVED** [review]
Looks good 👍

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**